schema/encoding/jsonschema/jsonschema.go

```diff
@@ -101,12 +101,20 @@ func validatorToJSONSchema(w io.Writer, v schema.FieldValidator) (err error) {
 		if t.ValuesValidator != nil {
 			ew.writeString(`, "items": `)
 			if ew.err == nil {
-				ew.err = validatorToJSONSchema(w, t.ValuesValidator)
+				ew.err = validatorToJSONSchema(w, t.ValuesValidator, true)
```

Comment:
why not.
ew.writeString("{")
ew.err = validatorToJSONSchema(w, t.ValuesValidator)
ew.writeString("}")

Then you don't need the extra object flag parameter...